R/class-RDAPlot.R

```diff
@@ -89,14 +104,20 @@ RDAPlot <- function(...) {
     new("RDAPlot", ...)
 }
 
-#' @importFrom iSEE .getEncodedName .selectInput.iSEE .numericInput.iSEE
+#' @importFrom iSEE .getEncodedName .selectInput.iSEE .numericInput.iSEE .sliderInput.iSEE
 #' @importFrom SingleCellExperiment reducedDimNames
 #' @importFrom methods slot
 setMethod(".defineDataInterface", "RDAPlot", function(x, se, select_info) {
     panel_name <- .getEncodedName(x)
     
     list(.selectInput.iSEE(x, field="dimred", label="Reduced dimension",
-        choices=reducedDimNames(se), selected=slot(x, "dimred")))
+            choices=reducedDimNames(se), selected=slot(x, "dimred")),
+        .sliderInput.iSEE(x, field="confidence.level", label="Confidence level",
```

Comment:
I would rather use an inputNumeric like the number of taxa to show in the AbundanceDensityPlot panel. This is because users probably want to have more flexible choice within 0.95 to 0.999 range than for the rest